judge/judgedaemon.main.php

```diff
@@ -275,16 +275,23 @@ function read_judgehostlog(int $n = 20) : string
 // Fetches new executable from database if necessary, and runs build script to compile executable.
 // Returns an array with absolute path to run script and possibly an error message.
 function fetch_executable(
-    string $workdirpath, string $type, string $execid, string $hash, int $judgeTaskId, bool $combined_run_compare = false) : array
-{
+    string $workdirpath,
+    string $type,
+    string $execid,
+    string $hash,
+    int $judgeTaskId,
+    bool $combined_run_compare = false
+) : array {
```

Comment:
The brace should go on a new line.

Reply:
Would that not take an extra unneeded line? I'll take a look why phpcs did not warn on this.